packages/teleport/src/Databases/AddDatabase/AddDatabase.story.test.tsx

```diff
@@ -0,0 +1,26 @@
+/*
+Copyright 2021 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import React from 'react';
+import { render } from 'design/utils/testing';
+import { Manually } from './AddDatabase.story';
+
+jest.mock('design/Modal/Modal', () => ({ children }) => children);
```

Comment:
i suggest going through the reading material i suggested in slack, and then dig into Modal and see where the modal is getting rendered